### DISCUSSION TITLE
Define multiple sources in Replication

### DISCUSSION THREAD

**Discussion by bjg90** [slingdata-io/sling-cli#437] [Q&A]
Hi,
I would like to use the replication config to define multiple postgres databases on the same pg server -> parquet file destinations, is there a way to do this within the same yaml file or do I need to create separate files for each source database? If separate can I name the yaml files differently?
Thanks

**Comment by flarco** [comment]
Hey, you would need to create different connections, each pointing to a specific database on the same server.
And subsequently, different yaml files (for each connection/database).
And yes, you can just name the file differently, as you wish.
